google-cloud-firestore/src/test/java/com/google/cloud/firestore/LocalFirestoreHelper.java

```diff
@@ -73,14 +73,23 @@
 import javax.annotation.Nullable;
 import org.mockito.invocation.InvocationOnMock;
 import org.mockito.stubbing.Answer;
+import org.threeten.bp.Duration;
 
 public final class LocalFirestoreHelper {
 
+  protected static RetrySettings IMMEDIATE_RETRY_SETTINGS =
+      RetrySettings.newBuilder()
+          .setInitialRetryDelay(Duration.ZERO)
+          .setMaxRetryDelay(Duration.ZERO)
+          .setRetryDelayMultiplier(1)
+          .setJittered(false)
+          .build();
+
   public static final String DATABASE_NAME;
   public static final String COLLECTION_ID;
   public static final String DOCUMENT_PATH;
   public static final String DOCUMENT_NAME;
-  public static final ByteString TRANSACTION_ID;
+  public static final String TRANSACTION_ID;
```

Comment:
What's the motivation for switching from `ByteString` -> `String` here and in the methods below?

Reply:
I moved the conversion to ByteString into the helper methods so that the API in this class accepts Strings.